### ISSUE TITLE
Taxonium Electron: Backend killed when loading tips and/or nextstrain json

### ISSUE THREAD

**Issue by corneliusroemer** [theosanderson/taxonium#414]
Tried downloading nextstrain json -> nothing happened

Tried listing tips -> after 5s white screen of death

Can't restart from this start screen again, it shows Axios error:
<img width="791" alt="image" src="https://user-images.githubusercontent.com/25161793/195361898-7c4f87ba-ceaa-4ba8-bb25-1c3154430098.png">

**Comment by corneliusroemer** [comment]
Ok so tested again, and tips work fine. Seems to be Nextstrain json download that's the problem. Not sure how "download" works in Electron.

**Comment by corneliusroemer** [comment]
Yep, reproducible: Nextstrain json download click causes white screen of death/crash

**Comment by theosanderson** [comment]
Interesting - I got NS download to work a few mins ago. Will look into it.

**Comment by theosanderson** [comment]
For info, to be able to see the backend logs, start the electron app in the terminal with `/Applications/Taxonium.app/Contents/MacOS/Taxonium`

**Comment by theosanderson** [comment]
Ooh yeah I just got the crash for the first time
